### PR TITLE
Search

### PR DESCRIPTION
**Pull request by acouch**
Adds basic search + loading changes

### CODE REVIEW

**Comment by acouch** [comment]
NOT READY :)

File: src/Home.js [hunk 1]
```diff
@@ -1,37 +1,50 @@
...
-          </div>
-        );
-    }
+	state = {
+    "items": [],
+		"loaded": false
+	}
+
+  async fetchData() {
+		const { data } = await backend.get(`/collections/organization.json`);
```
Comment (fmizzell) [inline src/Home.js:16]: We need to use the dkan_api endpoints instead of the interra_api endpoints.

File: src/Search.js [hunk 2]
```diff
@@ -1,61 +1,48 @@
...
+
+class Search extends Component {
+
+  state = {
+    "items": [],
+		"loaded": false
+  }
+
+  async fetchData() {
+    const { data } = await backend.get(`/search-index.json`);
```
Comment (fmizzell) [inline src/Search.js]: we should use the dkan_api endpoints.

File: src/components/HomePageIconList/TopicsList.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
```
Comment (fmizzell) [inline src/components/HomePageIconList/TopicsList.js]: If there are nay modifications to any of these components, the changes should be moved to data-catalog-components.

**Review by fmizzell — CHANGES_REQUESTED** [review]

File: src/Home.js (hunk 1 above)
Comment (acouch) [reply src/Home.js:16]: Why would we use different endpoints? The point here is we are not tied to DKAN. We are already implementing the Interra endpoints. We could update them or make them more compatible with data packages if that would work better. Lets talk on Tues.

File: src/components/HomePageIconList/TopicsList.js (hunk 3 above)
Comment (acouch) [reply src/components/HomePageIconList/TopicsList.js]: These changes were pushed w/ this PR: https://github.com/interra/data-catalog-components/pull/5/files

I think of this as a working branch that would need to remove these components before being acceptable to be pulled in.

File: src/Search.js (hunk 2 above)
Comment (acouch) [reply src/Search.js]: We discussed internally.

**Pull request merged**